gp2gp_adaptor_response_docs.yaml

```diff
@@ -0,0 +1,557 @@
+                skeleton:
+                    type: boolean
+        SentToMhs:
+            type: object
+            properties:
+                messageId:
+                    type: array
+                    items:
+                        type: string
+                sentAt:
+                    type: string
+                taskId:
+                    type: string
+        GpcAccessDocument:
+            type: object
+            properties:
+                documents:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/GpcDocument'
+                patientId:
+                    type: string
+        Error:
+            type: object
+            properties:
+                occurredAt:
+                    type: string
+                    format: date-time
+                code:
+                    type: string
+                    nullable: False
+                    description: 'The GP2GP response code from the negative response'
+                message:
+                    type: string
+                    nullable: False
+                    description: 'The GP2GP response text from the negative response'
+                taskType:
+                    type: string
+        EhrExtractStatus:
+            type: object
+            properties:
+                id:
+                    type: string
+                created:
+                    type: string
+                    format: date-time
+                updatedAt:
+                    type: string
+                    format: date-time
+                conversationId:
+                    type: string
+                ehrRequest:
+                    $ref: '#/components/schemas/EhrRequest'
+                gpcAccessStructured:
+                    $ref: '#/components/schemas/GpcAccessStructured'
+                gpcAccessDocument:
+                    $ref: '#/components/schemas/GpcAccessDocument'
+                ehrExtractCore:
+                    $ref: '#/components/schemas/EhrExtractCore'
+                ehrExtractCorePending:
+                    $ref: '#/components/schemas/EhrExtractCorePending'
+                ehrContinue:
+                    $ref: '#/components/schemas/EhrContinue'
+                ackToRequester:
+                    $ref: '#/components/schemas/AckToRequester'
+                ackPending:
+                    $ref: '#/components/schemas/AckPending'
+                ehrReceivedAcknowledgement:
+                    $ref: '#/components/schemas/EhrReceivedAcknowledgement'
+                error:
+                    $ref: '#/components/schemas/Error'
+                messageTimestamp:
+                    type: string
+                    format: date-time
+                ehrExtractMessageId:
+                    type: string
+                ackHistory:
+                    $ref: '#/components/schemas/AckHistory'
+            description: Details about EhrExtractStatus
```

Comment:
Does this schema definition get used anywhere? It also references schema types that are only referenced by this.

Reply:
I've removed those schemas